### PR TITLE
Add Percy tests for blog category

### PR DESCRIPTION
**Pull request by mmmavis**
Related PRs/issues #2516

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "build": "npm run build:common && npm run build:js",
     "build-uncompressed": "npm run build:common && npm run build:js-uncompressed",
     "cypress": "run-p --race server cypress:test",
-    "cypress:install": "npm install --no-save cypress@^3.1.5 @percy/cypress@^2.0.0",
+    "cypress:install": "npm install --no-save cypress@3.6.1 @percy/cypress@^2.0.0",
```
Comment (mmmavis) [inline package.json]: Is this good or we want a new PR to update Cypress to latest version?

File: cypress/integration/endpoint-tests.js [hunk 2]
```diff
@@ -62,6 +62,22 @@ describe(`Visual regression testing for foundation.mozilla.org`, () => {
...
+    cy.window()
+      .its(`main-js:react:finished`)
+      .should(`equal`, true);
+    cy.wait(500);
+    cy.percySnapshot();
+  });
+
+  // Skipping as currently there's a bug related to Cypress about
+  // it not being able to visit a url that redirects
+  // GitHub ticket: https://github.com/mozilla/foundation.mozilla.org/issues/4042
```
Comment (Pomax) [inline cypress/integration/endpoint-tests.js]: :+1:

File: package.json (hunk 1 above)
Comment (Pomax) [reply package.json]: this should be fine, but if we're bumping this, let's use 3.7.0 instead.

Comment (mmmavis) [reply package.json]: fixed

**Comment by mmmavis** [comment]
@Pomax our "master" copy of Percy snapshots are still catching up (I just merged the YouTube Regrets Percy test PR like 10 mins) that's why it shows there are two new snapshots being added by this PR.

image

**Comment by mmmavis** [comment]
woohooo

**Pull request merged**